### PR TITLE
Update OX FAQ/index.md

### PR DESCRIPTION
**Pull request by boyledan68**
Updated the folder cleanup FAQ.

### CODE REVIEW

**Review by bhazzard — APPROVED** [review]
Lookin good.

File: content/how-to/rackspace-email/open-xchange-faq/index.md [hunk 1]
```diff
@@ -34,10 +34,21 @@ No, the price you pay will remain the same.
 
 ### The following features are changing: 
 - The maximum forward address limit will change to 4 (Currently 15)
-- Folder cleanup: 
-	- We will implement a new global setting for the spam and trash folders.
-	- The settings for inbox, draft, and sent folders will be removed.
-	- You will be notified if your mailbox is affected by this change.
+- Folder cleanup:
```
Comment (ssalinas24) [inline content/how-to/rackspace-email/open-xchange-faq/index.md:37]: We probably need a different permalink that will be unique. At the moment we have this article and another article both using `rackspace-email-faq` for permalinks. Might just need to use `open-xchange-faq/`

Comment (boyledan68) [reply content/how-to/rackspace-email/open-xchange-faq/index.md:37]: Updated

**Review by bhazzard — APPROVED** [review]
looks good.

**Pull request merged**